lib/howl/commands/app_commands.moon

```diff
@@ -665,39 +668,112 @@ do_search = (search, whole_word) ->
       m = matches[i]
       file_search_hit_to_location(m, search, display_as)
 
-  locations, searcher, project
+  locations, searcher
 
-command.register
-  name: 'project-file-search',
-  description: 'Searches files in the the current project'
-  input: (opts) ->
-    editor = app.editor
-    search = nil
-    whole_word = false
+ask_for_search_directory = (search_term) ->
+  start_dir = get_buffer_dir(app.editor.buffer)
+  if not start_dir
+    start_dir = howl.io.File(howl.sys.env.HOME)
```

Comment:
Not required. If `select_directory` is not given a `path:`, it will fall back to the home directory anyway.